presto-native-execution/presto_cpp/presto_protocol/presto_protocol.yml

```diff
@@ -355,4 +355,6 @@ JavaClasses:
   - presto-main/src/main/java/com/facebook/presto/connector/system/SystemTableHandle.java
   - presto-main/src/main/java/com/facebook/presto/connector/system/SystemColumnHandle.java
   - presto-main/src/main/java/com/facebook/presto/connector/system/SystemTableLayoutHandle.java
-  - presto-main/src/main/java/com/facebook/presto/connector/system/SystemTransactionHandle.java
\ No newline at end of file
+  - presto-main/src/main/java/com/facebook/presto/connector/system/SystemTransactionHandle.java
```

Comment:
Would be a good idea to separate the protocol update into a separate commit. 

There has been a recent change to change protocol generation to use sorted order of classes. https://github.com/prestodb/presto/pull/23339. Please rebase with it.